lang/fr-fr.js

```diff
@@ -91,9 +91,9 @@ export default {
 	"components.more-less.less": "moins",
 	"components.more-less.more": "plus",
 	"components.overflow-group.moreActions": "Plus d'actions",
-	"components.pager-load-more.action": "Load {count} More",
-	"components.pager-load-more.info": "{showingCount} of {totalCount} items",
-	"components.pager-load-more.status-loading": "Loading more items",
+	"components.pager-load-more.action": "Charger {count} supplémentaire(s)",
```

Comment:
@dbatiste: French seems to be the only language that handled pluralization... wonder if we should have used the `plural` function here?

Reply:
Yeah, I think we should.  Oversight on my part.  I think I also need to properly format the number.  If you want I can create a separate PR to fix those things together and you can just accept this change knowing it will get updated.

Reply:
Ok sounds like a plan, thanks!